Lombiq.DataTables.Samples/Recipes/Lombiq.DataTables.Samples.Content.recipe.json

```diff
@@ -543,7 +543,7 @@
               "Text": "London"
             },
             "Age": {
-              "Value": 66
+              "Value": 65
```

Comment:
Why was this change necessary?

Reply:
As explained in [the commit message](https://github.com/Lombiq/Orchard-Data-Tables/pull/182/commits/b1b6f845d63fb8ed3dbabcf372ebf50dc2c2d1fd) sorting was ambiguous in the UI test, because we had two items with the largest age value. After the upgrade it kept picking sorting the other one first. Since they are equal, this is not technically wrong but it failed the test.